css/index.css

```diff
@@ -3,144 +3,336 @@
-    margin-bottom: 20px;
-    border-bottom: 1px dashed black;
+  margin-bottom: 20px;
+  border-bottom: 1px dashed black;
 }
 
 .middle-content h2 {
-    padding: 0 2%;
-    margin-bottom: 0;
+  padding: 0 2%;
+  margin-bottom: 0;
 }
 
 .middle-content .boxes {
-    display: flex;
-    flex-wrap: wrap;
-    justify-content: space-evenly;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
 }
 
 .middle-content .boxes .box {
-    width: 12.5%;
-    height: 100px;
-    background: black;
-    margin: 20px 2.5%;
-    color: white;
-    display: flex;
-    align-items: center;
-    justify-content: center;
+  width: 12.5%;
+  height: 100px;
+  /* background: black; */
+  margin: 20px 2.5%;
+  color: white;
+  display: flex;
+  align-items: center;
+  justify-content: center;
 }
 
 .bottom-content {
-    display: flex;
-    margin: 0 2% 20px;
-    justify-content: space-around;
+  display: flex;
+  margin: 0 2% 20px;
+  justify-content: space-around;
 }
 
 .bottom-content .text-container {
-    padding-right: 4%;
+  padding-right: 4%;
 }
 
 .bottom-content .text-container:last-child {
-    padding-right: 0;
+  padding-right: 0;
 }
 
 footer {
-    width: 100%;
-    background: black;
+  width: 100%;
+  background: black;
 }
 
 footer nav {
-    width: 60%;
+  width: 60%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 20px 2%;
+  font-size: 14px;
+}
+
+footer nav a {
+  color: white;
+  text-decoration: none;
+}
+
+.middle-content .box:nth-child(1) {
+  background: teal;
+}
+
+.middle-content .box:nth-child(2) {
+  background: gold;
+}
+
+.middle-content .box:nth-child(3) {
+  background: cadetblue;
+}
+
+.middle-content .box:nth-child(4) {
+  background: coral;
+}
+
+.middle-content .box:nth-child(5) {
+  background: crimson;
+}
+
+.middle-content .box:nth-child(6) {
+  background: forestgreen;
+}
+
+.middle-content .box:nth-child(7) {
+  background: darkorchid;
+}
+
+.middle-content .box:nth-child(8) {
+  background: hotpink;
+}
+
+.middle-content .box:nth-child(9) {
+  background: indigo;
+}
+
+.middle-content .box:last-child {
+  background: dodgerblue;
+}
+
+div .about {
+    /* border: 5px solid black; */
     display: flex;
     justify-content: space-between;
-    align-items: center;
-    padding: 20px 2%;
-    font-size: 14px;
+    /* padding: */
+    flex-wrap: wrap;
+    /* background: red; */
+    margin-bottom: 5%;
 }
 
-footer nav a {
-    color: white;
+.about section {
+    display: flex;
+    /* content:40%; */
+    /* flex-basis: 100px; */
+    width: 48%;
+    flex-direction: column;
+    margin-bottom: 2%;
+    padding-bottom: 2%;
+    border-bottom: 1px dashed black;
+}
+
+.button {
+    padding: 5px 10px;
+    border: 1px solid black;
     text-decoration: none;
-}
\ No newline at end of file
+    text-align: canter;
+    color: black;
+    border-radius: 10px;
+    margin-right: 70%;
+}
+
+.bottom-content {
+    display: flex;
+    flex-direction: column;
```

Comment:
It appears this class is being used in both the home and about page, which is causing conflict since home needs to be row and about needs to be column.
Try instead to have two different class names to avoid style clashes.